### ISSUE TITLE
Description or screenshots?

### ISSUE THREAD

**Issue by inlinecoder** [Dimillian/Musix#2]
Hey,

Can you please make a bit better description of what Musix does (is capable of)? Maybe attach a few screenshots?

I'm personally very interested in it, especially the user authentication to get a MusicKit user token to be supplied along with dev token to get personalized recommendations and play full track versions, not just previews.

**Comment by Dimillian** [comment]
For now it just displays the top charts, and you can search any albums, it's very fast ;) 
I'll provide screenshot later today... need to renew my token.

The problem tho, is that the CloudKit Music part is not available on macOs so you can't request a user token, and you can't stream any music..... maybe in the next version of the SDK...I hope.

**Comment by inlinecoder** [comment]
> The problem tho, is that the CloudKit Music part is not available on macOs so you can't request a user token, and you can't stream any music..... maybe in the next version of the SDK...I hope.

Yeah, that's the point :) I was thinking I'm the one who can't get it work.

Thanks for the response.

**Comment by inlinecoder** [comment]
@Dimillian man, I caught myself at thinking about how Apple made Apple Music app to work on Android? Do you think if it makes sense to see what requests are sent to make user logged in? Did you get the idea I'm talking about?

(I have a testing Android phone and Apple Music works there like a charm: sign in, offline downloads, and all the rest functionality like on iOS)

**Comment by Dimillian** [comment]
Wise, it would be good to put some Wireshark in between and see how it works. I don't think it yet use the new MusicKit API but at least we should be able to see how they request a stream a song. But I guess it would be near impossible to reverse engineer due to a lot of security measure they take. 

I'm sure next year we'll have a a streaming API, it makes sense.

**Comment by inlinecoder** [comment]
It definitely needs tinkering :) 
I wonder if it's enough to get a user-token to use with MusicKit or Apple Music REST API.
